oremda/pipeline/runner/rpc/client.py

```diff
@@ -0,0 +1,101 @@
+import uuid
+import asyncio
+from typing import Dict
+from fastapi_websocket_rpc import RpcMethodsBase, WebSocketRpcClient
+
+from oremda.typing import DisplayType, IdType, JSONType, PipelineJSON
+from oremda.pipeline.runner.context import GlobalContext
+from oremda.pipeline.messages import NotificationMessage, pipeline_created
+from oremda.pipeline.observer import ServerPipelineObserver
+from oremda.pipeline.displays import DisplayHandle1D, DisplayHandle2D
+from oremda.pipeline.models import PipelineModel, SerializablePipelineModel
+from oremda.pipeline import deserialize_pipeline
+from oremda.display import NoopDisplayHandle
+
+
+def unique_id():
+    return str(uuid.uuid4())
```

Comment:
Can we have this in a utils file? Maybe re-use the one in `oremda.utils.id`?

Reply:
Oops, that was the intention.